services/ui-src/src/styles/theme.ts

```diff
@@ -147,13 +147,23 @@ const theme = extendTheme({
           ...theme.components.Button.variants.transparent,
           border: "1px solid",
           borderColor: "palette.primary",
+          textDecoration: "none",
           _hover: {
             ...theme.components.Button.variants.transparent._hover,
             borderColor: "palette.primary_darker",
             span: {
               filter: svgFilters.primary_darker,
             },
           },
+          _visited: {
+            color: "palette.primary",
+          },
+          ":hover, :visited:hover": {
+            color: "palette.primary_darker",
+          },
+          _focus: {
+            textDecoration: "none",
+          },
```

Comment:
Author note: Just some needed additional theme styling to add to this button variant to make it stay cute while extending `Link` functionality.